packages/cli/tests/utils/compiler.spec.ts

```diff
@@ -20,13 +21,37 @@ describe('Compiler utils', async () => {
         global.window = {};
     });
 
-    describe('compile', () => {
+    describe('compile Block', () => {
         test('should provide a valid build with block and settings', async () => {
-            await compile(rootPath, pathToIndex, 'index');
+            await compileBlock({ projectPath: rootPath, entryFile: pathToIndex, outputName: 'index' });
             await import(outputFile);
             expect(global.window).toHaveProperty('index');
             expect(global.window?.index.block).toBe('this is a block');
             expect(global.window?.index.settings).toMatchObject({ some: 'settings' });
         });
     });
+
+    describe('compile PlatformApp', () => {
+        const testHash = 'mocked hash';
+        vi.mock('crypto', () => ({
```

Comment:
`vi.mock` is hoisted, so for clarity could you move the block below the imports?